doc/api/deprecations.md

```diff
@@ -2668,19 +2668,25 @@ The [`crypto.Certificate()` constructor][] is deprecated. Use
 ### DEP0147: `fs.rmdir(path, { recursive: true })`
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/37216
+    description: Permissive behavior was removed, the option is still deprecated.
   - version: v15.0.0
     pr-url: https://github.com/nodejs/node/pull/35562
-    description: Runtime deprecation.
+    description: Runtime deprecation for permissive behavior.
   - version: v14.14.0
     pr-url: https://github.com/nodejs/node/pull/35579
     description: Documentation-only deprecation.
 -->
 
-Type: Runtime
+Type: Documentation-only
 
-In future versions of Node.js, `fs.rmdir(path, { recursive: true })` will throw
+`fs.rmdir(path, { recursive: true })`, `fs.rmdirSync(path, { recursive:true })`
+and `fs.promises.rmdir(path, { recursive:true })` throws
 if `path` does not exist or is a file.
-Use `fs.rm(path, { recursive: true, force: true })` instead.
+Use `fs.rm(path, { recursive: true, force: true })`,
```

Comment:
This still mainly refers to the permissive behavior, which is removed in `v16.0.0`.

Wondering if it would get the point across better if we went with:

```md
`fs.rmdir(path, { recursive: true })`, `fs.rmdirSync(path, { recursive:true })` will be removed in the next major, use Use `fs.rm(path, { recursive: true, force: true })`, etc.
```

I'm 👍 with this decision as long as the TSC is supportive -- avoid keeping both API surfaces.

Reply:
I don't think we want to say it will be removed in 17.x. "a future version" is sufficient. "the next version" is more specific than we've been in the past, and we've been known to kick things to subsequent majors for various reasons.

Reply:
Oh, wait, it says "a future version" now. Cool.